src/nupic/bindings/algorithms.i

```diff
@@ -25,31 +25,9 @@
 %import <nupic/bindings/math.i>
 
 %pythoncode %{
-# ----------------------------------------------------------------------
```

Comment:
This header got moved to the common boilerplate from all .i modules. So, there is now only one place to maintain it.